pkg/input/controller/reconciler.go

```diff
@@ -37,15 +38,20 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 	obj := r.controlledObjectPrototype.DeepCopyObject().(client.Object)
 	obj.SetName(request.Name)
 	obj.SetNamespace(request.Namespace)
-	if err := client.IgnoreNotFound(r.client.Get(ctx, request.NamespacedName, obj)); err != nil {
-		return reconcile.Result{}, fmt.Errorf("reconcile: retrieve object from server: %w", err)
+
+	isObjectMissingOnServer := false
```

Comment:
```suggestion
	isObjectNotFound := false
```

Reply:
"isObjectNotFound" was an earlier take on the name. However, the double negative "not isObjectNotFound" is harder to read than "not isObjectMissingOnServer".

Reply:
My motivation for this comment that it is misleading to say that the object is missing in the server. The client is most likely a cached one (based on informers behind the scenes). Hence, it does not do a get request to the server on `Get()`. Instead, it is getting the object from the informer's cache. Hence, you don't really know what is the state on the server, you only fetch the object from the cache.